tests/test-cases/readers/cube-config/v1.0/cube_datatypes.json

```diff
@@ -118,8 +118,13 @@
             "label": "time attribute",
             "data_type": "time"
         },
+        "Attr-number": {
+            "type": "attribute",
+            "label": "number attribute",
+            "data_type": "double"
```

Comment:
Wasn't the data type here supposed to be `number`? I thought that was the whole point of the test?

Reply:
Yeah it is. Have changed